DMX.Core.Api/Models/Orchestrations/LabWorkflows/Exceptions/LabWorkflowOrchestrationValidationException.cs

```diff
@@ -0,0 +1,16 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using Xeptions;
+
+namespace DMX.Core.Api.Models.Orchestrations.LabWorkflows.Exceptions
+{
+    public class LabWorkflowOrchestrationValidationException : Xeption
+    {
+        public LabWorkflowOrchestrationValidationException(Xeption innerException)
+            : base(message: "Lab workflow orchestration validation error occured. Please fix and try again.",

```

Comment:
Don't reference the level in the end-user message. 
So in here we just say `Lab workflow validation error occurred.
Also occurred is two Cs and two Rs. always.

Do the same for the rest, please.